app/models/course/group.rb

```diff
@@ -27,8 +28,8 @@ def set_defaults
   # @return [Boolean]
   def should_create_manager?
     course && creator &&
-      course.course_users.exists?(user: creator) &&
-      !group_users.exists?(user: creator)
+      course.course_users.exists?(user_id: creator.id) &&
+      !group_users.map(&:course_user).map(&:user_id).include?(creator.id)
```

Comment:
Hmm okay. was thinking if we need `user` accessible through `group_user` - if not, what you suggested is 👍